mod/message.php

```diff
@@ -165,11 +165,23 @@ function message_content(App $a)
 
 		$cmd = $a->argv[1];
 		if ($cmd === 'drop') {
+			$r = DBA::SelectFirst('mail', ['convid'], ['id' => $a->argv[2], 'uid' => local_user()]);
+			if(!DBA::isResult($r)){
+				info(L10n::t('Conversation not founded.') . EOL);
+				goaway('/message');
+			}
+
 			if (DBA::delete('mail', ['id' => $a->argv[2], 'uid' => local_user()])) {
 				info(L10n::t('Message deleted.') . EOL);
 			}
 
-			goaway('/message' );
+			$rr = DBA::SelectFirst('mail', ['id'], ['convid' => $r['convid'], 'uid' => local_user()]);
+			if(!DBA::isResult($rr)){
+				info(L10n::t('Conversation removed.') . EOL);
+				goaway('/message');
+			}
+
+			goaway('/message/'.$rr['id'] );
```

Comment:
Standards: Please add spaces around operators, and remove space inside parentheses.